marbles/core/setup.py

```diff
@@ -62,7 +62,7 @@
             'flake8-per-file-ignores'
         ]
     },
-    python_requires='>=3.4,<3.7',
+    python_requires='>=3.4,<3.8',
```

Comment:
Shouldn't this be strictly greater than 3.4?

Reply:
oops